house_of_refuge/frontend/src/components/SubmissionRow.js

```diff
@@ -150,8 +150,14 @@ export function SubmissionRow({sub, activeHandler, user, isGroupCoordinator, isA
   return <div
       className={`submission-row ${getStatusClass(localSub)}
       ${localSub.accomodation_in_the_future ? "sub-in-future" : ""} ${isActive ? "sub-active" : ""}`}>
-    <p className="sub-id">ID ZGŁOSZENIA: {localSub.id}</p>
-    <Table className="sub-table">
+    <div className="sub-id position-relative">
+      ID ZGŁOSZENIA: {localSub.id}
+
+      <div className="submission-row-collapse" onClick={() => setCollapsed(!collapsed)}>
+        Zwiń / Rozwiń
+      </div>
+    </div>
+    <Table className="sub-table" style={{'background-color': 'rgba(255, 255, 255, 0.9)', display: collapsed ? 'none' : 'table'}}>
```

Comment:
I liked a bit more of contrast maybe bump 0.9 -> 0.95 ?

Reply:
Done